src/test/resources/http-request-status-codes-config.xml

```diff
@@ -19,18 +26,15 @@
     </flow>
 
     <flow name="success">
-        <http:request config-ref="requestConfig" path="status">
+        <http:request config-ref="requestConfigWithDefaultSuccessValidator" path="status">
```

Comment:
This looks like it's reducing the coverage, shouldn't you add tests here instead of override the existing ones?